lib/parsers/moov_parser/decoder.rb

```diff
@@ -1,6 +1,7 @@
 # Handles decoding of MOV/MPEG4 atoms/boxes in a stream. Will recursively
 # read atoms and parse their data fields if applicable. Also contains
 # a few utility functions for finding atoms in a list etc.
+# To know more about Atoms: https://developer.apple.com/library/archive/documentation/QuickTime/QTFF/QTFFChap2/qtff2.html
```

Comment:
I missed a link to the docs with more details about the Atoms. I added this one, but I'm not sure if it is the best/correct one.